src/index.js

```diff
@@ -1 +1,95 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+// Base URL for Leaderboard API
+const baseURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+// Function to handle API requests
+const sendRequest = async (url, method = 'GET', data = {}) => {
+  try {
+    const requestOptions = {
+      method,
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    };
+
+    // Add the body only for non-GET requests
+    if (method !== 'GET') {
+      requestOptions.body = JSON.stringify(data);
+    }
+
+    const response = await fetch(url, requestOptions);
+
+    if (!response.ok) {
+      throw new Error('Request failed.');
+    }
+
+    const result = await response.json();
+    return result;
+  } catch (error) {
+    // Display error message in feedback element
+    const feedback = document.getElementById('feedback');
+    feedback.innerHTML = `${error.message}`;
+  }
+
+  // To adhere to consistent-return
+  return null;
+};
+
+// Create a new game
+// const createGame = async (name) => {
+//   const url = baseURL + 'games/';
+//   const data = { name };
+
+//   const result = await sendRequest(url, 'POST', data);
+//   return result;
+// };
+
+// createGame('FGames');
```

Comment:
- [ ] Kindly remove all commented pieces of code from your documents. This will make your documents messy.